srcCpp/RTIDDSImpl.cxx

```diff
@@ -2829,6 +2811,25 @@ IMessagingReader *RTIDDSImpl<T>::CreateReader(
     }
 }
 
+std::string getQoSProfileName(const char *topicName)
+{
+    std::string qosProfile = std::string("");
```

Comment:
Regarding to create a map instead to check it with if/else.
The proper way is creating a
`static std::map<std::string,std::string> qoSProfileMap`
@javmorcas tell me if you like this.

Reply:
Lets keep it with this function for the time being.

Reply:
Done